.github/workflows/build_front.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build Frontend
+
+on:
+  push:
+    branches: 
+      - master
+      - dev
+      - v1
+  pull_request:
+    branches: 
+      - "*"
+
+jobs:
+  build:
```

Comment:
try to separate jobs ? build / lint / test ?

Reply:
be careful, jobs name must be not redundant between other actions file (for better understanding)